enterprise/server/raft/filestore/filestore.go

```diff
@@ -68,6 +68,16 @@ func fileRecordSegments(r *rfpb.FileRecord) (partID string, groupID string, isol
 
 type PebbleKeyVersion int
 
+const (
+	// PebbleKeyVersion representing an unspecified key version. When this is
+	// the active key version, the database will select an active key version
+	// to use based on existing database contents and the maximum known key
+	// version.
+	//
+	// This is in a different const block so as not to affect the iota below.
+	UnspecifiedKeyVersion PebbleKeyVersion = -1
+)
+
 const (
 	// UndefinedKeyVersion is the version of all keys in the database
```

Comment:
Do we still need this? What does this represent?

Reply:
I think this is the version of un-migrated keys from before version 0 and could exist for on-prem deployments that have never upgraded the version.